.devcontainer/Dockerfile

```diff
@@ -0,0 +1,2 @@
+FROM mcr.microsoft.com/devcontainers/python:1-3.11
+RUN apt update && apt install -y ffmpeg
```

Comment:
My experience (and conclusion after some brief googling) was that the easiest way of avoiding the stack trace error in the Home Assistant logs was to install `ffmpeg` in the devcontainer. While it could be done in the `"postCreateCommand": "./container dev-setup"` script, doing it in this Dockerfile speeds up container rebuilds as it leverages the `docker build` image layer caching. The downside of installing `ffmpeg` is that it adds some ~120MB to an image that was already quite large (~1.4GB).